cylc/flow/scripts/lint.py

```diff
@@ -617,7 +617,13 @@ def list_wrapper(line: str, check: Callable) -> Optional[Dict[str, str]]:
             for job_runner, directive in WALLCLOCK_DIRECTIVES.items()
         )),
         FUNCTION: check_wallclock_directives,
-    }
+    },
+    'S015': {
+        'short': (
+            '`=>` is a line continuation without `\\`.'
```

Comment:
```suggestion
            '`=>` implies line continuation without `\\`.'
```
(Pretty sure you accepted this change on the original!)